.github/workflows/update-server-side-sdk-schema.yml

```diff
@@ -94,7 +87,6 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: ${{ inputs.language-version }}
-          cache: 'pnpm'
```

Comment:
It's ok to have it, but let's install pnpm first

Reply:
Fixed!